src/tests/bulkConcepts/components/PreviewCard.test.js

```diff
@@ -38,7 +39,7 @@ describe('Test suite for ActionButton in BulkConceptsPage component', () => {
             to_concept_code: '',
           },
         ],
-        descriptions: [{ description: '' }],
```

Comment:
Is this change related to the ticket?

Reply:
@dkayiwa this was not supposed to be part of the changes. I have sorted it.